tests/test_cff2pages.py

```diff
@@ -302,32 +335,52 @@ def test_generated_current_html_body(self):
     </footer>
 </div>
 
-</body>""" # noqa
+</body>"""  # noqa
 
 
 def check_folders(cls, tmp_dir):
+    """
+    Tests if the temp_dir was created and have the necessary files in it.
+
+    :param cls: Test class which does the assertion process.
+    :param tmp_dir: Directory where the files are stored.
+    :return: Path to the html
+    """
     public_path = os.path.join(tmp_dir, 'public')
     index_file = os.path.join(public_path, 'cff2pages.html')
     cls.assertTrue(os.path.exists(public_path))
     cls.assertTrue(os.path.exists(index_file))
     return index_file
 
 
-class VersionTester(unittest.TestCase):
+class CffTomlComparer(unittest.TestCase):
+    """
+    Testing class to verify that the different aspects between toml and cff are the same.
```

Comment:
```suggestion
    Testing class to verify that the various aspects between toml and cff are the same.
```

Using a synonym here to avoid confusion from 'different' and 'the same' appearing in quick succession.